src/core/src/pass/serialize.cpp

```diff
@@ -51,14 +51,17 @@ struct Edge {
 // convention. Most of them are the same, but there are exceptions, e.g
 // Constant (OpenVINO Model name) and Const (IR name). If there will be more
 // discrepancies discovered, translations needs to be added here.
-const std::unordered_map<std::string, std::string> translate_type_name_translator = {{"Constant", "Const"},
+static const std::unordered_map<std::string, std::string>& get_translate_type_name_translator() {
```

Comment:
minor: we already in unnamed namespace, so `static` is not required

Reply:
fixed, thanks